package.json

```diff
@@ -13,7 +13,9 @@
     "packages/sdk-bridge",
     "packages/sdk-govern",
     "packages/indexer",
-    "packages/keymaster"
+    "packages/keymaster",
+    "packages/sdk-zodiac-nomad-module",
```

Comment:
can this name be shorter? 😅

Reply:
Totally.  Jury is still out on this. We were also discussing `sdk-governance`.  Also discussed possibly renaming `sdk-govern` to `sdk-core-governance` to avoid naming confusion. (see slack thread linked above)